src/submission/models.py

```diff
@@ -1273,6 +1273,16 @@ def active_reviews(self):
     def completed_reviews(self):
         return self.reviewassignment_set.filter(is_complete=True, date_declined__isnull=True)
 
+    @property
+    def completed_reviews_with_permission(self):
+        return self.reviewassignment_set.filter(is_complete=True, date_declined__isnull=True,
+                                                permission_to_make_public=True)
+
+    @property
+    def public_reviews(self):
```

Comment:
can be more DRY with `return self.completed.reviews_wih_permission.filter(display_public=True)`